app/controllers/return-logs-setup.controller.js

```diff
@@ -38,8 +47,25 @@ async function submitReceived(request, h) {
   return h.redirect(`/system/return-logs/setup/${sessionId}/reported`)
 }
 
+async function submitStart(request, h) {
+  const { sessionId } = request.params
+
+  const pageData = await SubmitStartService.go(sessionId, request.payload)
+
+  if (pageData.error) {
+    return h.view('return-logs/setup/start.njk', {
```

Comment:
Not a massive issue but to keep it consistent inside the controller maybe the activeNavBar property can go inside the SubmitStartService, like how the submitReceived function looks
```suggestion
    return h.view('return-logs/setup/start.njk', {...pageData})
```